src/TextToTalk/TextToTalk.cs

```diff
@@ -71,44 +72,57 @@ public TextToTalk([RequiredVersion("1.0")] DalamudPluginInterface pi)
             var framework = this.services.GetService<Framework>();
             framework.Update += PollTalkAddon;
             framework.Update += CheckKeybindPressed;
-            framework.Update += CheckPresetKeybindPressed;
 
             var commands = this.services.GetService<Dalamud.Game.Command.CommandManager>();
             this.commandManager = new CommandManager(commands, this.services);
             this.commandManager.AddCommandModule<MainCommandModule>();
+
+            this.commandModule = new MainCommandModule();
```

Comment:
This line should be removed; it incorrectly creates the object (none of its properties are injected by the service container) and the value is immediately overwritten anyways on line 81 (which is the correct way of fetching the existing instance).

Reply:
Ok, I originally tried initializing `this.commandModule` as an instance variable in the same way that you had previously initialized it as a local variable, but it gave a `System.Reflection.TargetInvocationException` immediately at runtime. It compiled fine though.

But I can maybe see why this could cause problems. I might just revert that whole commit since none of the other changes in that commit work right without that line 80 there.